src/components/hackathon/LaunchCard.tsx

```diff
@@ -0,0 +1,21 @@
+interface LaunchCardProps {
+	icon: JSX.Element;
+	text: string;
+	onClick: () => void;
+}
+
+export default function LaunchCard({ icon, text, onClick }: LaunchCardProps) {
```

Comment:
nothing to revise, just noting I like how u took in the props. Never thought you could do it that way! I will definitely be stealing that